platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDb.java

```diff
@@ -884,23 +885,24 @@ public MerkleDbTableConfig getTableConfig() {
         public int pbjSizeInBytes() {
             int size = 0;
             if (tableId != 0) {
-                size += ProtoUtils.sizeOfTag(FIELD_TABLEMETADATA_TABLEID, ProtoUtils.WIRE_TYPE_VARINT);
-                size += ProtoUtils.sizeOfVarInt32(tableId);
+                size += ProtoWriterTools.sizeOfTag(
+                        FIELD_TABLEMETADATA_TABLEID, ProtoConstants.WIRE_TYPE_VARINT_OR_ZIGZAG);
```

Comment:
Changing the wire type changes semantics.  Is this intended?

Reply:
There's not a change actually. The `ProtoUtils.WIRE_TYPE_VARINT` was defined as `0`:

https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/utilities/ProtoUtils.java#L30

and `ProtoConstants.WIRE_TYPE_VARINT_OR_ZIGZAG` is defined as 0 ordinal in the enum:

https://github.com/hashgraph/pbj/blob/main/pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/ProtoConstants.java#L6-L9

So the referenced wire type is exactly the same, and the new code is identical to the old one from the semantics perspective.

Reply:
Interesting.  The names imply different encoding (and the change wasn't mentioned), so I thought it worth asking.